src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Replace the literal `50` with a constant that represents the discount for long term rentals. Use a descriptive name like `LONG_TERM_DISCOUNT`.